modules/user_auth/files/kratos-values.yml

```diff
@@ -0,0 +1,156 @@
+image:
+  repository:  oryd/kratos
+  tag: v0.5.4-alpha.1-sqlite
+  pullPolicy: IfNotPresent
+
+ingress:
+  public:
+    enabled: true
+    annotations:
+        kubernetes.io/ingress.class: nginx
+        # nginx.ingress.kubernetes.io/rewrite-target: /$1
+        cert-manager.io/cluster-issuer: clusterissuer-letsencrypt-production
+    hosts: # Filled by user_auth.tf
+      - paths: ["/"]
+    tls:
+      - secretName: kratos-public-tls-secret
+
+  admin:
+    enabled: true
+    annotations:
+        kubernetes.io/ingress.class: nginx
+        # nginx.ingress.kubernetes.io/rewrite-target: /$1
+        cert-manager.io/cluster-issuer: clusterissuer-letsencrypt-production
+    hosts: # Filled by user_auth.tf
+      - paths: ["/"]
+    tls:
+      - secretName: kratos-admin-tls-secret
+
+# To set up OIDC you can uncomment this and create a secret which contains the
+# OIDC provider configuration in JSON format in an env var called SELFSERVICE_METHODS_OIDC
+# See README.md for more info
+# deployment:
+  # environmentSecretsName: secretName
+
+kratos:
+  development: false
+  autoMigrate: true
+
+#  You can add multiple identity schemas here
+  identitySchemas:
+    "identity.default.schema.json": |
```

Comment:
At some point we'll want to make sure people can pass this in, but it's probably fine like this for now.